src/Engines/webgpuEngine.ts

```diff
@@ -298,6 +298,9 @@ export class WebGPUEngine extends Engine {
         return Promise.reject("gslang is not available.");
     }
 
+    public getDevice(): GPUDevice {

```

Comment:
I would prefer to not expose any WebGPU specific types out of engine and keep all specific work enclosed inside. 

Is that possible ?